draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2941,21 +2942,30 @@ https://video.example.com/k/movie1/4/013.ts
 		</list>
 
 	</t>
+
+        <t>
+        uCDN may combine multiple options in the same trigger command. dCDN should consider the following when processing such commands:
+        <list style="symbols">
+        <t>dCDN MUST reject triggers that are dependent on other pending triggers with lower priority to prevent deadlocks.</t>
+        <t>When introducing an urgent trigger uCDN SHOULD indicate a priority that is equal to or higher than the highest priority among the pending triggers that belong to the same uCDN. dCDN MUST reject urgent triggers that have lower priority than other pending triggers of the same uCDN.</t>
+        <t>dCDN MUST reject an urgent trigger that has pending triggers as its prerequisite. dCDN MAY reject an urgent trigger that has triggers in “active” or “cancelling” state as its prerequisite, such as when such triggers are deemed to take a long time to complete or cancel.</t>
+        </list>
+        </t>
+
 	<t>
-	uCDN may combine multiple options in the same trigger command. dCDN should consider the following when processing such commands:
+	When combining ExtensionPolicy with <xref target="time-policy"/>, TimePolicy governs the time window when the trigger can execute, while ExecutionPolicy controls the order of trigger execution within their respective windows.  The following error conditions should be considered by dCDN:
 	<list style="symbols">
-	<t>(i) dCDN MUST reject triggers that are dependent on other pending triggers with lower priority to prevent deadlocks.</t>
-	<t>(ii) When introducing an urgent trigger uCDN SHOULD indicate a priority that is equal to or higher than the highest priority among the pending triggers that belong to the same uCDN. dCDN MUST reject urgent triggers that have lower priority than other pending triggers of the same uCDN.</t>
-	<t>(iii) dCDN MUST reject an urgent trigger that has pending triggers as its prerequisite. dCDN MAY reject an urgent trigger that has triggers in “active” or “cancelling” state as its prerequisite, such as when such triggers are deemed to take a long time to complete or cancel.</t>
+		<t>When a trigger is marked urgent, but TimePolicy doesn't allow immediate processing start, it should be rejected by dCDN</t>
+		<t>when a trigger with TimePolicy is dependent on another trigger with TimePolicy window (directly or via chain of dependency) that doesn't start until after the depending trigger's TimePolicy ends, such trigger should be rejected</t>
```

Comment:
when -> When
This might be easier to read if it is phrased "a TimePolicy".
Also, for consistency, perhaps we should change "TimePolicy window" to just "TimePolicy", since we know it is a window by definition.
Again, adding a period to the end will make it consistent.

Reply:
Here's what I came up with: 
                <t>When a trigger is marked urgent, but its TiimePolicy extension doesn't allow immediate processing, it should be rejected by dCDN.</t>
                <t>When a trigger has a TimePolicy extension and is also dependent on another trigger with a TimePolicy extension, that specifies an execution window (directly or via chain of dependency) that doesn't start until after the depending trigger's execution window ends, such trigger should be rejected.</t>


I think TimePolicy window reads better because a window can start and end, while it's not obvious that TimePolicy would.